### PR TITLE
OCM-1225 | fix: validate machinepool labels for /

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

File: cmd/create/machinepool/cmd.go [hunk 1]
```diff
@@ -232,6 +233,14 @@ func run(cmd *cobra.Command, _ []string) {
 	val, ok := cluster.Properties()[properties.UseLocalCredentials]
 	useLocalCredentials := ok && val == "true"
 
+	if cmd.Flags().Changed("labels") {
```
Comment (gdbranco) [inline cmd/create/machinepool/cmd.go:236]: Does it needs the same treatment on node pools?

Comment (davidleerh) [reply cmd/create/machinepool/cmd.go:236]: +1

Comment (thomasmckay) [reply cmd/create/machinepool/cmd.go:236]: There is an if-statement at the bottom of each of these files that directs to either machine- or node- pools. Which file(s) were you thinking also need changes?

Comment (gdbranco) [reply cmd/create/machinepool/cmd.go:236]: Oh if it comes later, that's alright then!

Comment (davidleerh) [reply cmd/create/machinepool/cmd.go:236]: Sorry, I thought that the labels were parsed / evaluated in the individual files as well (np vs mp) as well.
I think we can set this as resolved~

**Review by ciaranRoche — APPROVED** [review]

**Pull request merged**